src/Apps/Artist/Routes/Consign/Components/ArtistConsignHeader.tsx

```diff
@@ -46,3 +68,137 @@ export const ArtistConsignHeader: React.FC<ArtistConsignHeaderProps> = props =>
     </SectionContainer>
   )
 }
+
+/**
+ * Displays a random set of images for a given consignment artist.
+ *
+ * Some notes about implementation:
+ *
+ * 1) Due to responsiveness and SSR we need to dynamically scale the width and the
+ *    height of the image based on overall width of the container.
+ * 2) For this we use our `<ResponsiveImage>` component.
+ * 3) However, this component achieves responsivity by forcing padding within a
+ *    container and then applying a "fit" to a background image.
+ * 4) Therefore, conventional borders cannot be applied, and a CSS filter is
+ *    required. This is based upon pixel data, and border faked.
+ * 5) The pixel data comes from a second image, sitting below the first, with
+ *    its brightness turned to -100.
+ * 5) It's not perfect, and the layout needs to also account for variable
+ *    aspect ratios.
+ * 6) The positioning CSS is also tricky, so please forgive...
+ */
+const HeaderImages = (props: {
+  artworksByInternalID: Consign_artworksByInternalID
+}) => {
+  const getRandomImages = () => {
+    const [left, right] = sampleSize(props.artworksByInternalID, 2)
+
+    // Layout requires that we only use vertically oriented images
+    const widthExceedsHeight = ({ image }) => {
+      return image.resized.width > image.resized.height
+    }
+    if (widthExceedsHeight(left)) {
+      return getRandomImages()
+    }
+    if (widthExceedsHeight(right)) {
+      return getRandomImages()
+    }
+
+    return [left, right]
+  }
+
+  const [leftImage, rightImage] = getRandomImages()
+
+  return (
+    <HeaderImageContainer>
+      <Flex width="100%" justifyContent="space-between">
+        <LeftImage>
+          {/*
+            Create a fake border around image using pixel data and filter,
+            then darken it.
+          */}
+          <ResponsiveImage
+            src={leftImage.image.resized.url}
+            width={leftImage.image.resized.width}
+            style={{
+              transformOrigin: "center",
+              filter: "invert(100%) brightness(-100%)",
+              backgroundPosition: "left",
+            }}
+          />
+
+          {/*
+            Then shrink the second image slightly, which brings out the border
+            described above.
+          */}
+          <ResponsiveImage
+            src={leftImage.image.resized.url}
+            width={leftImage.image.resized.width}
+            style={{
+              position: "absolute",
+              top: 0,
+              left: -4,
+              transformOrigin: "center",
+              backgroundPosition: "left",
+              transform: "scaleX(.97) scaleY(.96",
```

Comment:
```suggestion
              transform: "scaleX(.97) scaleY(.96)",
```

I'm not sure what happens without a terminating parentheses, but I noticed one missing